modules/configure-ocp-oauth-identity-provider.adoc

```diff
@@ -25,25 +25,28 @@ To integrate the built-in {ocp} OAuth server as an identity provider for {produc
 For security, Red Hat recommends first setting the *Minimum access role* to *None* while you complete setup. Later, you can return to the *Access Control* page to set up more tailored access rules based on user metadata from your identity provider.
 ====
 
-. To add access rules for users and groups accessing {product-title-short}, click *Add new rule* in the *Rules* section. For example:
-.. To give the *Admin* role to a user called `administrator`, you can use the following key-value pairs to create access rules:
+. Optional: To add access rules for users and groups accessing {product-title-short}, click *Add new rule* in the *Rules* section, then enter the rule information and click *Save*. You will need attributes for the user or group so that you can configure access. 
 +
-|===
-| *Key* | *Value*
-|Name
-|administrator
-|Role
-|Admin
-|===
-.. If you are using the link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.11/html-single/authentication_and_authorization/index#configuring-htpasswd-identity-provider[HTPasswd Identity Provider] with the username `UserA` that is part of the group `GroupA`, you can use the following key-value pairs to create access rules:
+[TIP]
+====
+Group mappings are more robust because groups are usually associated with teams or permissions sets and require modification less often than users.
+====
++
+To get user information in {ocp}, you can use one of the following methods:
++
+* Use the link:https://docs.openshift.com/container-platform/4.11/rest_api/user_and_group_apis/user-user-openshift-io-v1.html[{ocp} API].
+* Click *User Management* -> *Users* -> *<username*> -> YAML.
+* Access the `k8s/cluster/user.openshift.io\~v1~User/<username>/yaml` file and note the values for `name`, `uid` (`userid` in {product-title-short}), and `groups`. 
 +
-|===
-| *Key* | *Value*
-|Name
-|UserA
-|Group
-|GroupA
-|UserID
-|<UUID>
-|===
-. Click *Save*.
+The following configuration example describes how to configure rules for an *Admin* role with the following attributes:
```

Comment:
The rendered version puts this paragraph under the last menu item of the list above, which is probably unintended. Same is true for the paragraph below.

Reply:
I've appealed for asciidoc help to my larger team since I can't figure out how to indent correctly :(

Reply:
I think we might have gotten it...it takes a village! :)